tests/t306-basis.c

```diff
@@ -2,6 +2,7 @@
 /// Test QR Factorization
 /// \test Test QR Factorization
 #include <ceed.h>
+#include <ceed-impl.h>
```

Comment:
I don't think this is needed. Otherwise I'm happy to squash+merge this.

Reply:
I'm not even sure when that was introduced.. I'll remove it